server.js

```diff
@@ -27,11 +35,68 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Start defining routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+/* // Get all songs - removed since I get all songs in next endpoint
+app.get('/songs', async (req, res) => {
+  const songs = await Song.find()
+  console.log(`Found ${songs.length} songs ...`)
+  res.json(songs)
+}) */
+
+// Get all songs: http://localhost:8080/songs
+// Get all feelgood songs: http://localhost:8080/songs?feelgood=true
```

Comment:
I think it's enough to have these example endpoints in your readme-file. Perhaps adding the possible routes to you're '/' endpoint as well?